src/Infusionsoft/Infusionsoft.php

```diff
@@ -226,7 +230,7 @@ public function requestAccessToken($code)
 
         $client = $this->getHttpClient();
 
-        $tokenInfo = $client->request($this->tokenUri, $params, array(), 'POST');
+        $tokenInfo = $client->request('POST', $this->tokenUri, ['body' => $params] );
```

Comment:
```Notice: Undefined index: headers in /infusionsoft-php/src/Infusionsoft/Http/GuzzleHttpClient.php on line 58```

This happens when doing the initial authorization request for the access token. Because the headers are null, it causes a fatal error.

```Catchable fatal error: Argument 3 passed to GuzzleHttp\Psr7\Request::__construct() must be of the type array, null given, called in /infusionsoft-php/src/Infusionsoft/Http/GuzzleHttpClient.php on line 58 and defined in /infusionsoft-php/vendor/guzzlehttp/psr7/src/Request.php on line 39```